### ISSUE TITLE
Pasting text misbehaves in Firefox

### ISSUE THREAD

**Issue by jakub-g** [Zolmeister/zethos#1]
Just tried to use zethos and while in Chrome I'm able to paste some text copied to the clipboard, in Firefox, the text is pasted somehow above/below the textarea and the application sees the data model in `undefined` state.

**Comment by mikermcneil** [comment]
Yeah the clipboard is the leather straps and contenteditable is a [giant hill of fire ants](https://medium.engineering/why-contenteditable-is-terrible-122d8a40e480#.8fegtap7z), crawling into your mouth.  That's a tough one to solve in < 1k.

But you could maybe just try to grab only the plain text if it's there- i.e. binding `paste` and then in the event handler:

```
var pastedPlainText = event.clipboardData.getData('text/plain');
```

That's what we're doing.
